compass/ocean/tests/internal_wave/rpe_test/namelist.forward

```diff
@@ -0,0 +1 @@
+config_run_duration = '20_00:00:00'
```

Comment:
Just a note to say that I agree with reducing the run duration from 100 days to 20 days.  The analysis is performed at 20 days so no added advantage in running 80 more days by default.  Also, the experiments in the paper were run for 200 days, so 100 days isn't consistent with the paper either.